### PR TITLE
[Updated] Sticky header , footer and namespace section in table viewb of scope

### PR DESCRIPTION
**Pull request by AVRahul**
Hover on the footer section to use help, debug and reset options.
hover1

Hover on the namespaces section to get all the namespaces which scrolls when more namespaces are added.
hover

Fixed the header so it will be sticky when scrolled up.
header

### CODE REVIEW

**Review by rake7h — APPROVED** [review]

File: client/app/styles/_base.scss [hunk 1]
```diff
@@ -241,7 +241,8 @@ a {
 
 .footer {
   @extend .overlay-wrapper;
-  right: 43px;
+  right: -380px;
+  bottom: -3px;
```
Comment (rake7h) [inline client/app/styles/_base.scss]: bottom -3px not require here.

File: client/app/styles/_base.scss [hunk 2]
```diff
@@ -873,6 +882,7 @@ a {
     tbody {
       position: relative;
 
+
```
Comment (rake7h) [inline client/app/styles/_base.scss]: check for sass-lint

File: client/app/styles/_base.scss (hunk 1 above)
Comment (AVRahul) [reply client/app/styles/_base.scss]: The bottom function is resolve from the footer section.

File: client/app/styles/_base.scss (hunk 2 above)
Comment (AVRahul) [reply client/app/styles/_base.scss]: the lint issues is resolved in scss file.

**Pull request merged**